docs/plugins/haveibeenpwned/plugin.js

```diff
@@ -0,0 +1,250 @@
+/**
+ * KeeWeb plugin: haveibeenpwned
+ * @author Olivier LEVILLAIN
+ * @license MIT
+ */
+
+const DetailsView = require("views/details/details-view");
+const Alerts = require("comp/alerts");
+const Logger = require('util/logger');
+const InputFx = require('util/input-fx');
+
+const detailsViewFieldChanged = DetailsView.prototype.fieldChanged;
+
+DetailsView.prototype.checkPwnedPwd = false;
+DetailsView.prototype.checkPwnedName = false;
+DetailsView.prototype.blockPwnedPwd = false;
+DetailsView.prototype.blockPwnedName = false;
+DetailsView.prototype.logger = new Logger("HaveIBeenPwned");
```

Comment:
It's better to put these variables as local, there's no point keeping them in DetailsView. Especially `logger`, which can replace a system logger, if it's there.

Reply:
changed to a settings local object